src/quo/components/internal_link_card/community/view.cljs

```diff
@@ -0,0 +1,101 @@
+(ns quo.components.internal-link-card.community.view
+  (:require
+    [quo.components.community.community-stat.view :as community-stat]
+    [quo.components.internal-link-card.community.style :as status-link-previews-style]
+    [quo.components.links.link-preview.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as theme]
+    [react-native.core :as rn]))
+
+(defn- description-comp
+  [description members-count active-members-count]
+  [rn/view
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     3
+     :accessibility-label :description} description]
+   [rn/view {:flex-direction :row :margin-top 12}
+    [community-stat/view
+     {:value               members-count
+      :icon                :i/members
+      :accessibility-label :members-count
+      :style               {:margin-right 12}}]
+    [community-stat/view
+     {:value               (or active-members-count 0)
+      :icon                :i/active-members
+      :accessibility-label :active-members-count}]]])
+
+(defn- title-comp
+  [title]
+  [text/text
+   {:size                :paragraph-1
+    :number-of-lines     1
+    :weight              :semi-bold
+    :style               style/title
+    :accessibility-label :title}
+   title])
+
+(defn- thumbnail-comp
+  [thumbnail]
+  [rn/image
+   {:style               status-link-previews-style/thumbnail
+    :source              (if (string? thumbnail)
+                           {:uri thumbnail}
+                           thumbnail)
+    :accessibility-label :thumbnail}])
+
+(defn- logo-comp
+  [logo]
+  [rn/image
+   {:accessibility-label :logo
+    :source              (if (string? logo)
+                           {:uri logo}
+                           logo)
+    :style               style/logo}])
+
+(defn- stat-loading
+  [theme]
+  [rn/view {:style status-link-previews-style/loading-stat-container}
+   [rn/view {:style (status-link-previews-style/loading-circle theme)}]
+   [rn/view {:style (status-link-previews-style/loading-stat theme)}]])
+
+(defn- loading-view
+  [theme]
+  [rn/view
+   {:accessibility-label :loading}
+   [rn/view {:style status-link-previews-style/row-spacing}
+    [rn/view (status-link-previews-style/loading-circle theme)]
+    [rn/view (status-link-previews-style/loading-first-line-bar theme)]]
+   [rn/view {:style (status-link-previews-style/loading-second-line-bar theme)}]
+   [rn/view status-link-previews-style/row-spacing
+    [stat-loading theme]
+    [stat-loading theme]]
+   [rn/view {:style (status-link-previews-style/loading-thumbnail-box theme)}]])
+
+(defn f-internal-view
+  [{:keys [title description loading? thumbnail-size icon banner members-count active-members-count
```

Comment:
@ibrkhalil - we have some guidelines to keep the component properties 1-1 with the figma properties. I think here it's missing this. Imo this makes it much easier to ensure all the variants are implemented properly and then to maintain them to as there is less second guessing of what variant we are creating. 
<img width="360" alt="Screenshot 2023-12-07 at 14 29 25" src="https://github.com/status-im/status-mobile/assets/22799766/1982de76-acae-4632-9a68-f09c066c6bb0">

Reply:
in fact it looks like this component might actually be more configurable than needs be.